kotlinx-coroutines-core/common/src/flow/terminal/Collect.kt

```diff
@@ -8,13 +8,26 @@
 package kotlinx.coroutines.flow
 
 import kotlinx.coroutines.*
+import kotlinx.coroutines.flow.internal.*
 import kotlin.jvm.*
 
+/**
+ * Terminal flow operator that collects the given flow but ignores all emitted values.
+ * If any exception occurs during collect or in the provided flow, this exception is rethrown from this method.
+ *
+ * It is a shorthand for `collect {}`.
+ *
+ * :todo: exception handling
+ */
+@ExperimentalCoroutinesApi
+public suspend fun Flow<*>.collect() = collect(NopCollector)
```

Comment:
Shouldn't this function be inline like `emitAll`?

Reply:
Making it inline does not add anything in terms of performance, but increases bytecode size for all users of this function.

Reply:
Oh, I just noticed `collect(…)` itself is `inline` too, I now understand how it'd make the bytecode bigger.